packages/fxa-admin-panel/src/components/PageAccountSearch/Account/index.tsx

```diff
@@ -0,0 +1,390 @@
+    if (!window.confirm('Are you sure? This cannot be undone.')) {
+      return;
+    }
+    await unlinkAccount({ variables: { uid } });
+
+    onCleared();
+  };
+
+  return (
+    <TableRowXHeader>
+      <>{providerId}</>
+      <>{getFormattedDate(authAt)}</>
+      <button
+        className="p-1 text-red-700 border-2 rounded border-grey-100 bg-grey-10 hover:border-2 hover:border-grey-10 hover:bg-grey-50 hover:text-red-700"
+        type="button"
+        onClick={handleUnlinkAccount}
+      >
+        Unlink
+      </button>
+    </TableRowXHeader>
+  );
+};
+
+export const Account = ({
+  uid,
+  email,
+  emails,
+  createdAt,
+  disabledAt,
+  locale,
+  lockedAt,
+  emailBounces,
+  totp: totps,
+  recoveryKeys,
+  attachedClients,
+  subscriptions,
+  onCleared,
+  query,
+  securityEvents,
+  linkedAccounts,
+}: AccountProps) => {
+  const createdAtDate = getFormattedDate(createdAt);
+  const disabledAtDate = getFormattedDate(disabledAt);
+  const lockedAtDate = getFormattedDate(lockedAt);
+  const primaryEmail = emails!.find((email) => email.isPrimary)!;
+  const secondaryEmails = emails!.filter((email) => !email.isPrimary);
+
+  const [editLocale] = useMutation(EDIT_LOCALE, {});
+  const handleEditLocale = async () => {
+    try {
+      const newLocale = window.prompt('Enter a new locale.');
+      if (!newLocale) {
+        return;
+      }
+
+      const res = await editLocale({
+        variables: {
+          uid,
+          locale: newLocale,
+        },
+      });
+
+      if (res.data?.editLocale) {
+        onCleared();
+      } else {
+        window.alert(`Edit unsuccessful.`);
+      }
+    } catch (err) {
+      window.alert(`An unexpected error was encountered. Edit unsuccessful.`);
+    }
+  };
+
+  function highlight(val: string) {
+    return query === val ? 'bg-yellow-100' : undefined;
+  }
+
+  return (
+    <>
+      <hr />
+      <section data-testid="account-section">
+        <TableYHeaders header="Account Details">
+          <TableRowYHeader
+            header="Sign-up Email"
+            value={<span className={highlight(email)}>{email}</span>}
+            testId="sign-up-email"
+          />
+          <TableRowYHeader
+            header="uid"
+            value={<span className={highlight(uid)}>{uid}</span>}
+            testId="account-uid"
+          />
+          <TableRowYHeader
+            header="Created At"
+            value={
+              <>
+                {createdAtDate} ({createdAt})
+              </>
+            }
+            testId="account-created-at"
+          />
+          <TableRowYHeader
+            header="Locale"
+            value={
+              <>
+                {locale}
+
+                <Guard features={[AdminPanelFeature.EditLocale]}>
+                  <button
+                    className="bg-grey-10 border-2 border-grey-100 font-small leading-6 ml-2 rounded text-red-700 w-10 hover:border-2 hover:border-grey-10 hover:bg-grey-50 hover:text-red-700"
```

Comment:
Possibly duped and unneeded styles:
- `text-red-700` and `hover:text-red-700`
- `border-2` and `hover:border-2`